src/runtime/README.md

```diff
@@ -23,66 +29,62 @@ The runtime is
  allowing it
 to work seamlessly with both Docker and Kubernetes respectively.
 
-## License
+## Download and install
 
-The code is licensed under an Apache 2.0 license.
-See [the license file](https://github.com/kata-containers/kata-containers/blob/main/LICENSE) for further details.
+[![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/kata-containers)
 
-## Platform support
+See the [installation guides](https://github.com/kata-containers/kata-containers/blob/main/docs/install/README.md)
+available for various operating systems.
 
-Kata Containers currently works on systems supporting the following
-technologies:
+## Architecture overview
 
-- [Intel](https://www.intel.com) VT-x technology.
-- [ARM](https://www.arm.com) Hyp mode (virtualization extension).
-- [IBM](https://www.ibm.com) Power Systems.
-- [IBM](https://www.ibm.com) Z mainframes.
-### Hardware requirements
+See the [architecture overview](../../docs/design/architecture)
+for details on the Kata Containers design.
 
-The runtime has a built-in command to determine if your host system is capable
-of running and creating a Kata Container:
+## Configuration
 
-```bash
-$ kata-runtime check
-```
+The runtime uses a TOML format configuration file called `configuration.toml`.
+The file is divided into sections for settings related to various
+parts of the system including the runtime itself, the [agent](../agent) and
+the [hypervisor](#hypervisor-specific-configuration).
+
+Each option has a comment explaining it use.
 
 > **Note:**
 >
-> - By default, only a brief success / failure message is printed.
-> If more details are needed, the `--verbose` flag can be used to display the
-> list of all the checks performed.
->
-> - `root` permission is needed to check if the system is capable of running
-> Kata containers. In this case, additional checks are performed (e.g., if another
-> incompatible hypervisor is running).
+> - The initial values in the configuration file provide a good default configuration.
```

Comment:
I think there shouldn't be a bullet for one element.

Reply:
Fixed.